docs/content/deploy.md

```diff
@@ -7,6 +7,7 @@ When you want to try out your game, you need to run a deployment of the game. Th
 
 Before you deploy your game, you need to build its workers; see documentation on [building workers]({{urlRoot}}/content/build.md).
```

Comment:
Please search and delete all superfluous instances of `.md` in improbadoc urls.

Reply:
Now fixed.